pkg/minishift/registration/redhat.go

```diff
@@ -77,8 +78,14 @@ func (registrator *RedHatRegistrator) Register(param *RegistrationParameters) er
 			}
 			// request password (disallow empty value)
 			if param.Password == "" {
+				param.Password = param.GetPasswordKeyring(param.Username)
+
 				for param.Password == "" {
 					param.Password = param.GetPasswordInteractive(RHSMName + " password")
+					err := param.SetPasswordKeyring(param.Username, param.Password)
+					if glog.V(3) {
+						fmt.Println(err)
```

Comment:
Gotca so here we are just print the error if log mode is 3 👍

Reply:
Right..